02-Reproject-Raster-In-R.Rmd

```diff
@@ -11,8 +11,8 @@ category: spatio-temporal-workshop
 tags: [raster-ts-wrksp, raster]
```

Comment:
The differences between the files were not appearing before. Rather, all i saw in the lesson was an entirely new file as if you rewrote the entire thing. However that was not the case. Whatever you did, it seems to show a diff now. I have a feeling the new file name did something. Don't worry about it - it's there now. Thank you.